contracts/masset/Masset.sol

```diff
@@ -787,39 +560,39 @@ contract Masset is
      * @return b   Basket struct
      */
     function getBasket() external view override returns (bool, bool) {
-        return (basket.undergoingRecol, basket.failed);
+        return (data.basket.undergoingRecol, data.basket.failed);
     }
 
     /**
      * @dev Get data for a all bAssets in basket
      * @return personal  Struct[] with full bAsset data
-     * @return data      Number of bAssets in the Basket
+     * @return bData      Number of bAssets in the Basket
      */
     function getBassets()
         external
         view
         override
-        returns (BassetPersonal[] memory personal, BassetData[] memory data)
+        returns (BassetPersonal[] memory personal, BassetData[] memory bData)
     {
-        return (bAssetPersonal, bAssetData);
+        return (data.bAssetPersonal, data.bAssetData);
```

Comment:
If this was called in a transaction, it could be more gas efficient to have
```
personal = data.bAssetPersonal;
bData = data.bAssetData;
```
But this will mostly be static called so probably doesn't matter.

Reply:
Yeah this is only called externally